product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -107,6 +107,58 @@ def test_pricelist_min_quantity(self):
             self.pricelist.get_product_price(self.product, 10, False), 50,
         )
         self.pricelist.item_ids[0].no_supplierinfo_min_quantity = True
+        # As the sequence of both of the supplieinfo is the same the next field to
+        # order is the min_qty (desc), so the price must be the one with the higher
+        # min_qty no mather if the qty is lower that the min_qty.
+        self.assertAlmostEqual(
+            self.pricelist.get_product_price(self.product, 1, False), 50,
+        )
+        self.assertAlmostEqual(
+            self.pricelist.get_product_price(self.product, 5, False), 50,
+        )
+
+    def test_pricelist_no_min_quantity(self):
+        """Testing the validations of the `_select_seller_without_min_qty` method.
+        As the sequence of both of the supplieinfo in the product is the same the
```

Comment:
```suggestion
        As the sequence of both supplierinfos in the product is the same, the
```

Also leave a blank line between title and description (it's the [docstring convention](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)).

Reply:
Done